### PR TITLE
[ADD] general: multi companies documentation

### PR DESCRIPTION
**Pull request by kbr-odoo**
[ADD] general: multi companies documentation

### CODE REVIEW

File: general/multi_companies/manage_multi_companies.rst [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+A centralized management environment allows you to select multiple companies simultaneously, and set
+their specific warehouses, customers, equipment and contacts. It provides you the ability to
+generate reports of aggregated figures without the need of switching interfaces, which facilitates
+daily tasks and the overall management process.
+
+Manage companies and records
+----------------------------
+
+Go to :menuselection:`Settings --> Manage Companies` and fill in the form with your company’s
+information. If a *Parent Company* is selected, records are shared between the two companies.
```
Comment (Lau-Odoo) [inline general/multi_companies/manage_multi_companies.rst:13]:
```suggestion
information. If a *Parent Company* is selected, records are shared between the two companies.
```

Not sure, but I think that records from the child company are accessible by the parent users but not the opposite, right?
(Or it's completely the opposite haha).

If I'm right, I think that it should be said here :)

File: general/multi_companies/manage_multi_companies.rst [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+- :doc:`Taxes <../../accounting/fiscality/taxes/default_taxes>`
+- :doc:`Fiscal Localizations <../../accounting/fiscal_localizations/overview/fiscal_localization_packages>`
+- :doc:`Pricelists <../../sales/products_prices/prices/pricing>`
+- :doc:`Warehouses <../../inventory/management/warehouses/warehouse_creation>`
+
+.. LINK JOURNAL AND FISCAL POSITIONS DOCS ONCE AVAILABLE + DELETE NOTE.
+
+Now, activate the *Inter-Company Transactions* option under *Settings*. With the respective company
+*activated* and *selected*, choose if you would like operations between companies to be synchronized
+at an invoice/bills level, or, at a sales/purchase orders level.
```
Comment (Lau-Odoo) [inline general/multi_companies/manage_multi_companies.rst:96]:
```suggestion
Now, activate the *Inter-Company Transactions* option under *Settings*. With the respective company
*activated* and *selected*, choose if you would like operations between companies to be synchronized
at an invoice/bills level or at a sales/purchase orders level.
```

File: general/multi_companies/manage_multi_companies.rst [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+
+.. image:: media/inter_company_transactions.png
+   :align: center
+   :alt: View of the settings page emphasizing the inter company transaction field in Odoo
+
+- **Synchronize invoice/bills**: generates a bill/invoice when a company confirms a bill/invoice for
+  the selected company.
+
+  *Example:* an invoice posted on JS Store Belgium for JS Store US, automatically creates a vendor
+  bill on the JS Store US from the JS Store Belgium.
```
Comment (Lau-Odoo) [inline general/multi_companies/manage_multi_companies.rst:106]:
```suggestion
  *Example:* an invoice posted on JS Store Belgium for JS Store US automatically creates a vendor
  bill on the JS Store US from the JS Store Belgium.
```

File: general/multi_companies/manage_multi_companies.rst (hunk 1 above)
Comment (kbr-odoo) [reply general/multi_companies/manage_multi_companies.rst:13]: Good one. So, records are accessible from both environments as long as both environments are active. Example: a record created in a "child" company can be seen by the parent, and vice-versa, as long as both dbs are activated.

:) thanks!

File: general/multi_companies/manage_multi_companies.rst (hunk 3 above)
Comment (kbr-odoo) [reply general/multi_companies/manage_multi_companies.rst:106]: I will keep the comma to give readers a break-time to absorb the 'on' and 'from' info ^^ (:

Comment (Lau-Odoo) [reply general/multi_companies/manage_multi_companies.rst:107]: Yeah, no problem :)
Then, can I suggest this:

```suggestion
  *Example:* an invoice posted on JS Store Belgium, for JS Store US, automatically creates a vendor
  bill on the JS Store US, from the JS Store Belgium.
```

File: general/multi_companies/manage_multi_companies.rst (hunk 1 above)
Comment (Lau-Odoo) [reply general/multi_companies/manage_multi_companies.rst]: Thanks for the explanation! :+1:

**Pull request merged**